### PR TITLE
Update fonts and other minor styling updates

### LINKED ISSUE #3: Fix Typekit fonts
The Typekit font kit we're referencing in the app no longer exists. Need to create a new kit and then update the JS code to refer to the appropriate Typekit URL.

### PR DESCRIPTION
**Pull request by ggeisler**
At some point we lost the typekit fonts, so this pull request adds back the typekit fonts. There are also a handful of other minor styling updates.

Fixes #3 

bassi-veratti_collection

bassi-veratti_collection 2

### CODE REVIEW

File: config/deploy.rb [hunk 1]
```diff
@@ -1,10 +1,11 @@
-lock '3.4.0'
+# config valid only for Capistrano 3.1
+lock '3.2.1'
```
Comment (cbeer) [inline config/deploy.rb]: Was this change intentional?

Comment (ggeisler) [inline config/deploy.rb]: @peetucket Can you address @cbeer's question (that line changed in your commit) so we can merge this? Thanks.

**Comment by ggeisler** [comment]
The `lock '3.2.1'` change to `config/deploy.rb` seems to affect my ability to deploy to -prod:

> $ cap production deploy
(Backtrace restricted to imported tasks)
cap aborted!
Capfile locked at 3.2.1, but 3.4.0 is loaded

Do I need to do something locally to load 3.2.1 rather than 3.4.0?

**Comment by cbeer** [comment]
I don't know why it was changed to `3.2.1` in the first place. If it wasn't intentional, I'd back out all those `config/deploy.rb` changes.

**Comment by ggeisler** [comment]
@peetucket Can I revert the `lock '3.2.1'` change in deploy.rb?  Is that part of your updates necessary?

File: config/deploy.rb (hunk 1 above)
Comment (peetucket) [inline config/deploy.rb]: not intentional - you can revert, not sure why (perhaps I deployed and have cap 3.4 in my system and needed to do that)
